### DISCUSSION TITLE
What happens if I don't use eslint-enable after eslint-disable?

### DISCUSSION THREAD

**Discussion by yamilelias** [eslint/eslint#16799] [Q&A] [answered]
I've been googling around and searching the discussions and issues but found nothing.

My question is: if I have this code for example:

```typescript
import * from 'my-awesome-package'

/* eslint-disable */
function sum(a: number, b: number) {
  return a + b
}
/* eslint-enable */

export default sum
```

What happens if I don't add the `/* eslint-enable */` in the file? Can it add some nasty bugs? Or is _eslint_ smart enough to detect this, and it won't matter?

I wanted to know because I've seen that some developers forget to use it, so I would love to give a strong argument for using it (more than sanity).

**Comment by mdjermanovic** [comment, marked as answer]
> What happens if I don't add the `/* eslint-enable */` in the file?

All errors and warnings reported in the range from `/* eslint-disable */` to the end of the file will be disabled.

If you want to make sure that every `eslint-disable` has an `eslint-enable`, you can use `eslint-comments/disable-enable-pair` rule:

https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/disable-enable-pair.html
